src/classes/groundplaces.ts

```diff
@@ -80,15 +81,16 @@ export class GroundPlacesController {
           return;
         }
 
-        if (isFilterByStopGroupActive && place.type !== GroundPlaceType.GROUP) {
-          return;
-        }
-
-        if (isFilterByStopClusterActive && place.type !== GroundPlaceType.CLUSTER) {
-          return;
+        // Return the place earlier if there is no filters to use
+        if (!filters || !filters.length) {
```

Comment:
I don't think this one is useful : no filter means nothing active means we proceed

Reply:
This is the step before the filters, if we don't have any filters, we don't need to pass through the filters checking above in the code. This is way I return the place here, I think this can save some time, don't you think?